src/main/java/ladder/generator/Generator.java

```diff
@@ -0,0 +1,12 @@
+package ladder.generator;
+
+import ladder.domain.ladder.Ladder;
+import ladder.domain.ladder.LadderHeight;
+import ladder.domain.ladder.Line;
+
+public interface Generator {
+
+    Ladder generate(int countOfPlayer, LadderHeight height);
+    Line generate(int countOfPlayer);
+
+}
```

Comment:
제네릭을 쓴다면, 그리고 Supplier 인터페이스를 더 활용한다면 제너레이터가 간결해질 수 있어요
```suggestion
public interface Generator<T, E> {
    T generate(E);
}
```